librad/src/profile.rs

```diff
@@ -0,0 +1,112 @@
+// Copyright © 2019-2021 The Radicle Foundation <hello@radicle.foundation>
+//
+// This file is part of radicle-link, distributed under the GPLv3 with Radicle
+// Linking Exception. For full terms see the included LICENSE file.
+
+use std::{env, fs, io, path::Path};
+use uuid::Uuid;
+
+use crate::paths::{project_dirs, Paths};
+
+/// A [`Profile`] provides [`Paths`] scoped by an identifier.
+///
+/// Profiles with different identifiers have distinct paths.
+#[derive(Debug, Clone)]
+pub struct Profile {
+    id: String,
+    paths: Paths,
+}
+
+impl Profile {
+    /// Creates a profile by loading the the profile identifier and paths from
+    /// the environment variables or well-known file.
+    ///
+    /// If `RAD_PROFILE` is not set, the value is used as the profile identifier
```

Comment:
Should this be, "If `RAD_PROFILE` is set"? Otherwise, what "value" are you referring to?

Reply:
Yes, of course! I’ve fixed this.